### PR TITLE
Remove comment and error from back when CodeWarrior was used

### PR DESCRIPTION
**Pull request by AtariDreams**
We aren't using CodeWarrior anymore. Let's fixup the function definition to match documentation and do struct assignment.

### CODE REVIEW

**Comment by Kyle-Ye** [comment]
@swift-ci please test

File: CoreFoundation/Base.subproj/CFUUID.c [hunk 1]
```diff
@@ -173,25 +173,8 @@ CFUUIDRef CFUUIDCreate(CFAllocatorRef alloc) {
...
-    bytes.byte7 = byte7;
-    bytes.byte8 = byte8;
-    bytes.byte9 = byte9;
-    bytes.byte10 = byte10;
-    bytes.byte11 = byte11;
-    bytes.byte12 = byte12;
-    bytes.byte13 = byte13;
-    bytes.byte14 = byte14;
-    bytes.byte15 = byte15;
+CFUUIDRef CFUUIDCreateWithBytes(CFAllocatorRef alloc, UInt8 byte0, UInt8 byte1, UInt8 byte2, UInt8 byte3, UInt8 byte4, UInt8 byte5, UInt8 byte6, UInt8 byte7, UInt8 byte8, UInt8 byte9, UInt8 byte10, UInt8 byte11, UInt8 byte12, UInt8 byte13, UInt8 byte14, UInt8 byte15) {
```
Comment (xwu) [inline CoreFoundation/Base.subproj/CFUUID.c:176]: Where is this `UInt8` type defined?

Comment (AtariDreams) [reply CoreFoundation/Base.subproj/CFUUID.c:176]: MacTypes.h

**Comment by GameCubeGBA** [comment]
Update?

**Comment by AtariDreams** [comment]
Update?

**Comment by AtariDreams** [comment]
@parkera Any updates?

**Comment by parkera** [comment]
Closing older PRs after re-core of swift-corelibs-foundation on swift-foundation (#5001).

**Pull request closed without merging**